templates/telecommunications/index.html

```diff
@@ -20,6 +20,9 @@ <h1>Ubuntu for telcos</h1>
         </ul>
       </p>
     </div>
+    <div class="col-4">
+      <img src="{{ ASSET_SERVER_URL }}49b9302b-network-equipment-white.svg" width="200" alt="">
```

Comment:
If this image is only going to be a maximum of 200. Could you add `?w=200` to the image src please.